packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -408,10 +415,12 @@ export function createNumericYAxis(
   // maxOfYVal coming from only area chart and Grouped vertical bar chart(Calculation done at base file)
   const tempVal = maxOfYVal || yMinMaxValues.endValue;
   const finalYmax = tempVal > yMaxValue ? tempVal : yMaxValue!;
-  const finalYmin = yMinMaxValues.startValue < yMinValue ? 0 : yMinValue!;
+  const finalYmin = supportNegativeValuesForYAxis
+    ? Math.min(yMinMaxValues.startValue, yMinValue || 0)
+    : Math.max(Math.min(yMinMaxValues.startValue, yMinValue || 0), 0);
```

Comment:
>Math.max(Math.min(yMinMaxValues.startValue, yMinValue || 0), 0); [](http://example.com/codeflow?start=6&length=64)

use the original logic when supportNegativeValues is false

Reply:
okay